include/common_structs.h

```diff
@@ -2059,18 +2135,45 @@ typedef struct SaveGlobals {
     /* 0x40 */ s8 reserved[64]; // unused
 } SaveGlobals; // size = 0x80
 
-typedef struct SaveMetadata {
+typedef struct FileDisplayData {
     /* 0x00 */ s32 timePlayed;
     /* 0x04 */ u8 spiritsRescued;
     /* 0x05 */ char unk_05[1];
     /* 0x06 */ s8 level;
     /* 0x07 */ unsigned char filename[8];
     /* 0x0F */ char unk_0F[9];
-} SaveMetadata; // size = 0x18
+} FileDisplayData; // size = 0x18
+
+typedef struct VanillaSaveData {
+    /* 0x0000 */ char magicString[16]; /// "Mario Story 006" string
+    /* 0x0010 */ char version[32]; /// always zero for vanilla saves
```

Comment:
This is incorrect Doxygen syntax. [docs](https://www.doxygen.nl/manual/docblocks.html#memberdoc)

You're meant to use `///<` but VS Code seems to not understand that syntax. So maybe we do
```suggestion
    /// "Mario Story 006" string
    /* 0x0000 */ char magicString[16];
    /// Always zero for vanilla saves
    /* 0x0010 */ char version[32];
```